### DISCUSSION TITLE
Problem with verifying custom domain

### DISCUSSION THREAD

**Discussion by kevando** [community/community#50772] [Other Feature Feedback, Questions, & Ideas]
I love Github Pages and I use it to host several websites on custom domains. However this time when I tried, I got this message

`The custom domain mason.money is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain.`

I followed those steps and got my domain verified, but I still get the same error when I try changing the domain on my Repo settings. What can I do?

https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages 

![bug](https://user-images.githubusercontent.com/1735712/227100458-8441f9ca-9122-483d-bf82-092fcd005623.png)

**Comment by Bihanso** [comment]
I met the same bug 🙉

**Reply by kevando** [reply]
What TLD are you using? I think my problem has to do with the `.mason` ending. I just did the same steps with with a `.com` and it worked right away.

**Comment by kevando** [comment]
mmmm... not exactly sure what's going on here but I think it's the exact thing they want to prevent.

my domain http://mason.money now loads a very spammy website and I have no idea where it came from. It looks like github pages so I guess it's cause I already set the DNS records for the A and CNAME 

lol I mean look at this site

![Screenshot 2023-03-24 at 11 08 40 PM](https://user-images.githubusercontent.com/1735712/227695543-6f8336b4-d04d-44de-be8e-39a98e456314.png)

**Comment by ersaope** [comment]
If you have already verified ownership of the domain and released it from any previous GitHub Pages configurations, but are still getting the "custom domain is already taken" error, there may be a few possible reasons:

DNS changes may take time to propagate: DNS changes can take some time to propagate, so it's possible that GitHub Pages is still detecting the domain as being in use by a previous configuration. It's recommended to wait for a few hours or up to 24 hours for DNS changes to fully propagate.

Caching issues: Caching issues can also cause GitHub Pages to still detect the domain as being in use. You can try clearing your browser cache or trying a different browser to see if that resolves the issue.

GitHub support: If neither of the above solutions work, you may need to contact GitHub support for further assistance. They may be able to help you resolve the issue or provide additional information on why the domain is still being detected as in use.

It's worth noting that GitHub Pages supports only one repository per custom domain, so if you have another repository already using the same custom domain, you won't be able to use that domain for another repository.

**Comment by kevando** [comment]
thanks @ersaope !!!

I had not considered that pre-emptively creating the DNS records that point to github might be causing an issue. I removed those records and no longer getting the "domain is already taken" error.

Thanks!

**Reply by PDillis** [reply]
If anyone has the same issue, it happened to me as well: I just redid all the DNS records (A, CNAME, TXT) since these existed before but I was just switching the domain name, so deleting them and redoing them was the trick. Thanks!

**Comment by robux111** [comment]
Hi,
I’m faced with a weird situation where Github Pages complains that my domain ysoftware.se is already taken.

The story is that weeks ago I have created a repository, enabled Github Pages with mrrfan.xyz as a custom domain, worked fine, but realized I should have actually rename a github org instead of creating a new one and moving repos. So I have deleted the github pages repo (uncertain if i removed the custom domain first) and the organization.

Now, whenever I want to enable the custom domain, I get: The custom domain [mrrfan.xyz](http://mrrfan.xyz/) is already taken. Check out Troubleshooting custom domains and GitHub Pages - GitHub Docs for more information.

I have verified that mrrfan.xyz organization owns mrrfan.xyz and I’ve also verified the domain for github pages. I have even tried to push manually a CNAME file but I get the same error via mail.

Since I own mrrfan.xyz I can actually set up the A records etc so that github pages serves mrrfan.xyz, but when I request mrrfan.xyz I only get a 404 page from github . So it’s really pointing into the void, and not to some repository’s actual github pages’ content.

I contacted support last week, got a first reply that my DNS is not set up correctly, but that’s obviously not what the issue at hand is. It has been silent since then. I upgraded my account to Team, not sure if that escalates somehow my ticket, but 🤞.

While waiting for an update from github support, I was wondering if others faced similar issues, and after searching the support community, I see that a handful of people were as unlucky as I am, but it looks like many/all cases have been manually handled by github staff. So here I am pleading my case 🙏

Thanks in advance,

**Reply by tuhinmukherjee** [reply]
Did the support team help you out with the issue? I’m having the same issue.
